Tests/FlowTests/OAuth2ExchangeAccessTokenForResourceTests.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  OAuth2ExchangeAccessTokenForResourceTests.swift
+//  OAuth2
+//
+//  Created by Tomas Ondrejka on 17/11/23.
+//  Copyright 2023 Dominik Paľo
```

Comment:
The correct copyright is Cisco Systems:


```suggestion
//  Copyright 2023 Cisco Systems, Inc. All rights reserved.
```

Reply:
Added 😄. I got a reference from `OAuth2DeviceGrantTests.swift`, which also has the `Copyright 2023 Dominik Paľo` signature. Should I also change this? @DominikPalo